internal/codes/random.go

```diff
@@ -1,17 +1,44 @@
 package codes
 
-import "crypto/rand"
+import (
+	"crypto/rand"
+	"math/big"
+)
 
 const (
-	codeSize    = 12
-	codeCharset = "346789BCDFGHJKMPQRTVWXY"
+	accessCodeSize            = 12                        // unformatted
+	paperVerificationCodeSize = 19                        // formatted
+	codeCharset               = "346789BCDFGHJKMPQRTVWXY" // pragma: allowlist secret
+	codeCharsetLength         = int64(len(codeCharset))
 )
 
-func randomCode() string {
-	bytes := make([]byte, codeSize)
-	_, _ = rand.Read(bytes)
```

Comment:
I wanted to change from this method as it isn't uniform, now it works in the same manner as the python